server/routes/bookVisit/additionalSupportController.test.ts

```diff
@@ -46,6 +50,17 @@ describe('Additional support needs', () => {
 
       app = appWithAllRoutes({ sessionData })
     })
+    it('should use the session validation middleware', () => {
```

Comment:
Added a similar test to each `/book-visit` route to be sure the validation middleware actually being called.